### PR TITLE
Plane param doc

### PR DESCRIPTION
**Pull request by WickedShell**
Fixes stuff from #3436 (Crash detect was correct I was tired when I was reading the code the first night). No functional changes.

### CODE REVIEW

File: ArduPlane/Parameters.cpp [hunk 1]
```diff
@@ -75,7 +75,7 @@ const AP_Param::Info Plane::var_info[] = {
     // @Description: bitmask of PIDs to send MAVLink PID_TUNING messages for
     // @User: Advanced
     // @Values: 0:None,1:Roll,2:Pitch,4:Yaw
-    // @Bitmask: 0:Roll,1:Pitch,2:Yaw
+    // @Bitmask: 0:Roll,1:Pitch,2:Yaw,4:Yaw
```
Comment (tridge) [inline ArduPlane/Parameters.cpp]: that doesn't look right ...

Comment (magicrub) [inline ArduPlane/Parameters.cpp]: double yaw?

Comment (WickedShell) [inline ArduPlane/Parameters.cpp]: Argh! Your right I should not look at that when tired...

**Comment by WickedShell** [comment]
Sorry about that, I read it wrong the other night, and just went at it with my recollection from the previous night, might as well just rename it to be a sugar cleanup on a single GPS param that causes some unneeded spaces that won't be trimmed when creating the parameter xml....

File: libraries/AP_GPS/AP_GPS.cpp [hunk 2]
```diff
@@ -80,7 +80,7 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
     // @Param: INJECT_TO
     // @DisplayName: Destination for GPS_INJECT_DATA MAVLink packets
     // @Description: The GGS can send raw serial packets to inject data to multiple GPSes.
-    // @Values: 0:send to first GPS, 1:send to 2nd GPS, 127:send to all
+    // @Values: 0:send to first GPS,1:send to 2nd GPS,127:send to all
```
Comment (magicrub) [inline libraries/AP_GPS/AP_GPS.cpp]: I think this "@Values" format breaks the stuff @DonLakeFlyer is doing

Comment (WickedShell) [inline libraries/AP_GPS/AP_GPS.cpp]: Having the spaces or not having them? We are terribly inconsistent, and it
made me add strip()'s to everywhere before parsing.

Comment (magicrub) [inline libraries/AP_GPS/AP_GPS.cpp]: Nevermind, it was @ Range he was fixing per https://github.com/diydrones/ardupilot/pull/3330

**Comment by tridge** [comment]
applied, thanks!

**Pull request closed without merging**